collect_app/src/main/java/org/odk/collect/android/preferences/screens/AdminPreferencesFragment.java

```diff
@@ -76,17 +76,17 @@ public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
 
         findPreference("project_name").setOnPreferenceChangeListener((preference, newValue) -> {
             Project currentProject = currentProjectProvider.getCurrentProject();
```

Comment:
This can just use `copy` right?

Reply:
Ok I did that but it's java, not kotlin (where I can update one specific parameter) so does that really matter?

Reply:
Hah I'd become so used to reading Kotlin I didn't notice. Yeah doesn't make sense to `copy` then.

Reply:
It's even less readable in java right? I think I should remove this change...

Reply:
Ok I removed that change I don't like it in java.